packages/aws-client/tests/test-S3.js

```diff
@@ -169,6 +163,35 @@ test('getS3Object() will retry if the requested key does not exist', async (t) =
   t.is(response.Body.toString(), 'asdf');
 });
 
+test('getObject() returns an existing S3 object', async (t) => {
```

Comment:
Consider moving these tests to `tests/S3/test-getObject.js`

Reply:
Will do.